SR2024/2023-09-27-place-rejection.md

```diff
@@ -0,0 +1,19 @@
+---
+to: Student Robotics 2024 rejections
+subject: Student Robotics 2024 is sadly full
+---
+
+Hello!
+
+We had a lot more interest in Student Robotics 2024 than we had places available. This has meant that unfortunately we're unable to offer you a place this year.
+
+We realise this is disappointing (we're disappointed too!), however we are limited by the number of kits we have and by the size of the venue. We are always looking for ways to support more teams and hope to be able to do so in future.
+
+If we have any early drop outs, we will be sure to let you know as you may be able to take their place, but we cannot promise anything.
+
+We hope you follow the competition on social media, and subscribe to our [mailing list](https://studentrobotics.org/compete/) to be the first to know when places open for Student Robotics 2025.

```

Comment:
Do we want to invite them to the competition?
Do we want to ask them to follow on social media?

Reply:
Invite to competition: Maybe? Though I'm erring towards not.
Social Media: Feels like asking for followers/subscribers could be a tad tacky in this context